docker-compose-tor.yml

```diff
@@ -0,0 +1,67 @@
+version: "3.7"
+
+services:
+  # TDEX daemon connected to Blockstream.info explorer
+  tdexd:
+    container_name: "tdexd"
+    image: ghcr.io/tdex-network/tdexd:latest
+    restart: unless-stopped
+    environment:
+      - TDEX_LOG_LEVEL=4 # 5 for Debug. 4 for production
+      - TDEX_CRAWL_INTERVAL=5000 # 5 seconds to poll blockchain
+      - TDEX_CRAWL_LIMIT=5000
+      - TDEX_CRAWL_TOKEN=500
+      - TDEX_EXPLORER_ENDPOINT=${EXPLORER} # 5 seconds to poll blockchain
+      
+      #### Macaroons auth/TLS termination - Operator interface ####
+      # Uncomment to disable macaroons auth
+      #- TDEX_NO_MACAROONS=true
+
+      # Uncomment one or both if running on a remote host with auth enabled
+      #- TDEX_OPERATOR_EXTRA_IP=0.0.0.0
+      #- TDEX_OPERATOR_EXTRA_DOMAIN=your.dns.domain
+      
+      #### TLS only - Trade interface ####
+      # Uncomment also the volumes down that mount to this paths
+      #- TDEX_SSL_CERT=./trade.crt
+      #- TDEX_SSL_KEY=./trade.key
+    ports:
+      - "9945:9945"
+      - "9000:9000"
+    #### Unlocker with file ####
+    # Uncomment to override default command and start tdexd and unlockerd.
+    # You need to uncomment also the volume down that mount the path used here.
+    # command: /bin/bash -c "(sleep 1 && unlockerd --password_path=/pwd.txt --interval=10 >& /dev/null &) && /tdexd-linux"
+      
+    volumes:
+      - ./tdexd:/home/user/.tdex-daemon
+      #### TLS only ####
+      #- ${SSL_CERT_PATH}:/trade.crt
+      #- ${SSL_KEY_PATH}:/trade.key
+      #### Unlocker with file ####
+      #- ${PWD_PATH}:/pwd.txt
+
+  # Pull prices from kraken and push to the daemon
+  feederd:
+    container_name: "feederd"
+    image: ghcr.io/tdex-network/feederd:latest
+    restart: unless-stopped
+    depends_on:
+      - tdexd
+    volumes:
+      - ./feederd/config.json:/config.json
```

Comment:
how we can pass the IP address of the machine and pass to this config JSON?

Reply:
Since it will be part of our automation, I guess one of the steps should be 
 **_dig TXT +short o-o.myaddr.l.google.com @ns1.google.com_**
 So with that command, we get the IP of the machine and we insert it into config, with sed or somehow :)

Reply:
nice...we should maybe add this bash script to source the IP of the machine and find&replace in the config.json also in the readme (for who uses without terraform automation)

Reply:
PS: feeder with latest commit in master refactored the config.json, so take a look